README.md

```diff
@@ -1 +1,12 @@
-# becode_database_api
\ No newline at end of file
+# becode_database_api
+
+CREATE DATABASE IF NOT EXISTS DB_notes;
+
+CREATE TABLE IF NOT EXISTS DB_notes.notes (
+  noteID int PRIMARY KEY AUTO_INCREMENT,
```

Comment:
zou 'int' en 'varchar' niet in hoofdletters moeten?

Reply:
> zou 'int' en 'varchar' niet in hoofdletters moeten?

not necessary